rules/kube_apiserver-config.libsonnet

```
@@ -8,9 +8,9 @@
     // These are buckets that exist on the apiserver_request_sli_duration_seconds_bucket histogram.
     // They are what the Kubernetes SIG Scalability is using to measure availability of Kubernetes clusters.
     // If you want to change these, make sure the "le" buckets exist on the histogram!
-    kubeApiserverReadResourceLatency: '1',
-    kubeApiserverReadNamespaceLatency: '5',
-    kubeApiserverReadClusterLatency: '30',
-    kubeApiserverWriteLatency: '1',
+    kubeApiserverReadResourceLatency: '1(\\.0)?',
+    kubeApiserverReadNamespaceLatency: '5(\\.0)?',
+    kubeApiserverReadClusterLatency: '30(\\.0)?',
+    kubeApiserverWriteLatency: '1(\\.0)?',
```

Escape characters are not required here, this should fix the lint issues:

```suggestion
    kubeApiserverReadResourceLatency: '1(.0)?',
    kubeApiserverReadNamespaceLatency: '5(.0)?',
    kubeApiserverReadClusterLatency: '30(.0)?',
    kubeApiserverWriteLatency: '1(.0)?',
```